includes/orphaned_objects.inc

```diff
@@ -0,0 +1,262 @@
+<?php
+
+/**
+ * @file
+ * A list of orphaned Islandora objects.
+ */
+
+/**
+ * Builds the Orphaned Islandora Objects management form.
+ *
+ * @param array $form
+ *   An array representing a form within Drupal.
+ * @param array $form_state
+ *   An array containing the Drupal form state.
+ *
+ * @return array
+ *   An array containing the form to be rendered.
+ */
+function islandora_manage_orphaned_objects_form($form, $form_state) {
+  if (isset($form_state['show_confirm'])) {
+    $pids = $form_state['pids_to_delete'];
+    if (count($form_state['pids_to_delete']) <= 10) {
+      $pid_paths = '<ul>';
```

Comment:
Rather than building the list as an html string it is best practice to use a render array to facilitate modification of the form. You may find the theme `item_list` useful.

Reply:
I couldn't figure out how to render an array within the 'confirm_message' array text; hence the string. Any tips?

Reply:
Not sure if it's correct, but a `#markup` element with `#theme => 'item_list'` and `#items` containing the items seems to do the trick. 
```php
  $pids = $form_state['pids_to_delete'];
    $form['confirm_message'] = array(
      '#type' => 'item',
      '#markup' => format_plural(count($pids),
      'Are you sure you want to delete this object? This action cannot be reversed.',
      'Are you sure you want to delete these @count objects? This action cannot be reversed.'),
    );
    if (count($pids) <= 10) {
      $form['pids_to_delete'] = array(
        '#type' => 'markup',
        '#theme' => 'item_list',
        '#list_type' => 'ol',
      );
      $options = array('attributes' => array('target' => '_blank'));
      foreach ($pids as $pid) {
        $form['pids_to_delete']['#items'][] = l($pid, "/islandora/object/{$pid}", $options);
      }
    }
```

Reply:
Thanks, @rosiel! That code works; I think it's all sorted now.

Reply:
Tried it, and it works! Thanks.